starport/services/chain/chain.go

```diff
@@ -182,16 +184,31 @@ func (c *Chain) appVersion() (v version, err error) {
 	if err != nil {
 		return version{}, err
 	}
-	iter, err := repo.Tags()
+	tags, err := repo.TagObjects()
 	if err != nil {
 		return version{}, err
 	}
-	ref, err := iter.Next()
+
+	var lastTag *object.Tag = nil
+
+	err = tags.ForEach(func(t *object.Tag) error {
+		if t == nil {
+			return errors.New("nil Tag exist in the TagObjects")
+		}
+
+		lastTag = t
```

Comment:
this method does not return the last tag because the `TagObjects` objects is unsorted

Reply:
Thank you for comment. In my case, I've just test on Mac OS and the result was OK.  
Anyway, I wrote the code again, and the link to the content is as follows. Can you check it out again?

https://gist.github.com/fly33499/6fb49e30f2e73744ed2bf3ebb3e87030